vllm/model_executor/layers/quantization/fbgemm_fp8.py

```diff
@@ -145,5 +146,5 @@ def apply(self,
                                 input_scale=None,
                                 input_scale_ub=layer.input_scale_ub,
                                 bias=bias,
-                                cutlass_fp8_supported=True,
+                                cutlass_fp8_supported=cutlass_fp8_supported(),
```

Comment:
can you save this as a variable in `__init__` its a bit expensive to call this on the hotpath

Reply:
Done